internal/rpchelp/helpdescs_en_US.go

```diff
@@ -316,6 +316,14 @@ var helpDescsEnUS = map[string]string{
 	// SignRawTransactionResult help.
 	"signrawtransactionresult-hex":      "The resulting transaction encoded as a hexadecimal string",
 	"signrawtransactionresult-complete": "Whether all input signatures have been created",
+	"signrawtransactionresult-errors":   "Script verification errors (if exists)",
+
+	// SignRawTransactionError help.
+	"signrawtransactionerror-error":     "Verification or signing error related to the input",
+	"signrawtransactionerror-sequence":  "Script sequence number",
+	"signrawtransactionerror-scriptSig": "The hex-encoded signature script",
+	"signrawtransactionerror-txid":      "The hash of the referenced, previous transaction",
+	"signrawtransactionerror-vout":      "The index of the output to spent and used as input",
```

Comment:
I'm not sure what this is trying to say.  Maybe:

"The output index of the referenced previous output"

?

And then we should change the one above to match:

"The transaction hash of the referenced previous output"